openshift/ftest/src/test/resources/arquillian.xml

```diff
@@ -4,7 +4,7 @@
     xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
 
 	<extension qualifier="openshift">
-		<property name="definitionsFile">src/test/resources/hello_pod.json</property>
+		<property name="env.config.url">src/test/resources/hello_pod.json</property>
```

Comment:
This is an example of a case, where we want to use `definitionsFile` and `env.config.url` separate. If we collapse them into a signle property, then we loose all the cubecontroller stuff. 

So I think that my idea about collapsing them, was not so good. Sorry for that.